src/api/comments.js

```diff
@@ -0,0 +1,7 @@
+import { get, post, remove } from './api';
+
+export const getPostComments = () => get('comments');
+export const createPostComment = (postId, name, email, body) => (
+  post('comments', {postId, name, email, body})
+);
+export const deletePostComment = (id) => get(`comments/${id}`);
```

Comment:
Are you sure that here should be `get` function?
```suggestion
export const deletePostComment = (id) => get(`comments/${id}`);
```
Why then you have imported `remove` function from './api'?)